src/Example.re

```diff
@@ -1,3 +1,5 @@
+// In your code you should use open ReactNavigation; to expose Stack.Make
+
```

Comment:
```suggestion
// Uncomment this to compile this example outside of this repo
// in this example it's not necessary (since we are running it in the module repo itself)
open ReactNavigation;
```